src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -137,13 +137,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $validAssetDirs = [];
         /** @var BundleInterface $bundle */
         foreach ($kernel->getBundles() as $bundle) {
-            if (!method_exists($bundle, 'getPublicDir')) {
-                @trigger_error(sprintf('Not defining "getPublicDir()" method in the "%s" class is deprecated since Symfony 4.4 and will not be supported in 5.0.', \get_class($bundle)), E_USER_DEPRECATED);
-                $publicDir = 'Resources/public';
+            if (method_exists($bundle, 'getBundleDir') && is_dir($bundle->getBundleDir().'/public')) {
+                $originDir = $bundle->getBundleDir().'/public';
             } else {
-                $publicDir = ltrim($bundle->getPublicDir(), '\\/');
```

Comment:
the ltrim path normalization got lost, potentially creating duplicate dir separators

Reply:
If we don't want the normalization, we need to document that `getPublicDir` is expected to return the path without the trailing directory separator. this is enough to me but it should be part of the phpdoc.

Reply:
That method `getPublicDir` doesn't exist in this PR, do you meant `getBundleDir()`?

Like `getPath()`, `getBundleDir()` returns an absolute path managed by `\dirname($reflected->getFileName());` which doesn't add the trailing directory separator, we're confident of it when we use `getPath()` everywhere without check it out.

Still, is it worth mentioning that in the phpdoc block?